velox/exec/MergeJoin.cpp

```diff
@@ -352,6 +354,28 @@ bool MergeJoin::addToOutput() {
   return outputSize_ == outputBatchSize_;
 }
 
+namespace {
+vector_size_t firstNonNull(
+    const RowVectorPtr& rowVector,
+    const std::vector<column_index_t>& keys,
+    vector_size_t start = 0) {
+  for (auto i = start; i < rowVector->size(); ++i) {
```

Comment:
`bits::findFirstBit` would be more efficient if this slows things down

Reply:
@Yuhta Thank you for the suggestion. I don't believe this is the hot spot in the merge join, but will keep in mind when we get to optimizing it.